### PR TITLE
Add "to the top" button

### PR DESCRIPTION
**Pull request by yovic4**
image

add the to the top button. for now can be found on "/" and "/sound-board". Will go to the top smoothly when clicked

### CODE REVIEW

File: packages/frontend/components/ToTheTopButton/ToTheTopButton.module.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@import 'styles/variables';
+
+.container {
+  display: grid;
+  justify-content: center;
+}
+
+.button {
+  display: grid;
```
Comment (RedMaps) [inline packages/frontend/components/ToTheTopButton/ToTheTopButton.module.scss]: (i know this is nagging on the highest level)
```suggestion
.button {
  cursor: pointer;
  display: grid;
```

File: packages/frontend/components/ToTheTopButton/ToTheTopButton.module.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  grid-template-columns: auto auto;
+  align-items: center;
+
+  color: $white;
+  background: #373737;
+  height: 2.4em;
+  width: 8.4em;
+  border-radius: 10px;
+
+  font-family: Roboto;
```
Comment (RedMaps) [inline packages/frontend/components/ToTheTopButton/ToTheTopButton.module.scss]: use the default font that was specified with the latest merge. I accidentally used roboto in the designs i think.
```suggestion
```

**Review by RedMaps — CHANGES_REQUESTED** [review]
This looks really good! Just two small things to change.

**Comment by yovic4** [comment]
Ok done

**Review by RedMaps — APPROVED** [review]
Well done!

**Pull request merged**